l10n_it_withholding_tax/views/report_invoice.xml

```diff
@@ -4,8 +4,34 @@
   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
   -->
 <odoo>
+    <template id="report_invoice" inherit_id="account.report_invoice">
```

Comment:
In https://github.com/odoo/odoo/blob/d573af55daca3e9e840aaaddaf8d75937b205391/addons/l10n_au/views/report_invoice.xml hanno fatto tutto ciò perché usa `replace` quindi non volevano rischiare di rompere altre viste; nel nostro caso credo si possa evitare perché non usiamo `replace`.
Ad esempio sarebbe meno invasivo togliere `primary` da `document_tax_totals_wt`, così non è più necessario modificare il `t-call` che solleva errore, cosa ne pensi?

Reply:
Non mi è chiaro, come funzionerebbe?

Reply:
> Non mi è chiaro, come funzionerebbe?

https://github.com/eLBati/l10n-italy/pull/110

Reply:
grazie, merged